lib/pages/nav_home_page.dart

```diff
@@ -12,7 +12,7 @@ class NavHomePage extends StatefulWidget {
 
 class _NavHomePageState
     extends State<NavHomePage> {
-  int _selectedIndex = 0;
+  int _selectedIndex = 1;
   String _appBarName = "Home";
 
   static const TextStyle optionStyle = TextStyle(
```

Comment:
This optionStyle doesn't seem to be used anywhere - can it be removed? I don't think it is something you've done but might as well get cleaned up now.